knex/knexfile.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
   production: {
     client,
     connection: process.env.DATABASE_URL,
+    ssl: { rejectUnauthorized: false },
```

Comment:
had to add this because we are using the free tier that doesn't support ssl... I think